test/test_grad_loss.jl

```diff
@@ -135,7 +135,20 @@ function test_grad_finite_diff(
     end
 
     dθ = zero(θ)
-    loss_iceflow_grad!(dθ, θ, simulation)
+    ######
+    # Computation of the gradient with SciMLSensitivity can fail with a fresh REPL
+    # Running the same code a second or third time usually works
+    # More information in https://github.com/ODINN-SciML/ODINN.jl/issues/354
+    try
+        loss_iceflow_grad!(dθ, θ, simulation)
+    catch
+        try
+            loss_iceflow_grad!(dθ, θ, simulation)
+        catch
+            loss_iceflow_grad!(dθ, θ, simulation)
+        end
+    end
+    ######
```

Comment:
I think is good to have this, but I will add a big warning if this executes under the `catch` statement just to provide some information about the error